server/cmwell-docs/cmwell-infodocs/Tools.UsingCM-WellDocker.md

```diff
@@ -39,7 +39,7 @@ To run CM-Well Docker, you must install the Docker utility itself and the relate
 >**Note:** Optimally, CM-Well Docker should have 6GB of RAM configured. If there is not enough available memory on your machine, CM-Well Docker will still run with 4.8GB of RAM, though not at production-level performance.
 
 ## Installing the CM-Well Image ##
-2. Download the CM-Well image tar file from [http://cm-well.int.thomsonreuters.com:8080/docker/cm-well-docker.tgz](http://cm-well.int.thomsonreuters.com:8080/docker/cm-well-docker.tgz).
+2. Download the CM-Well image tar file from coming soon.
```

Comment:
Naomi will perform a full review of all documentation and will take care of it in the future.